packager/packager.go

```diff
@@ -338,20 +339,39 @@ func (this *Packager) ServeHTTP(resp http.ResponseWriter, req *http.Request, par
 		}
 	}()
 
-	if !this.shouldPackage() {
+	if !this.shouldPackage() || !amp_cache_transform.ShouldSendSXG(req.Header.Get("AMP-Cache-Transform")) {
 		proxy(resp, fetchResp)
 		return
 	}
 
 	switch fetchResp.StatusCode {
+	case 200:
```

Comment:
Ah, I missed doing this in my last commit. Thanks for cleaning up.

Reply:
No prob!